Open Judge System/Web/OJS.Web/Areas/Administration/ViewModels/Problem/DetailedProblemViewModel.cs

```diff
@@ -148,5 +177,12 @@ public string SolutionSkeletonShort
         }
 
         internal byte[] SolutionSkeletonData { get; set; }
+
+        public override Problem GetEntityModel(Problem model = null)
+        {
+            model = model ?? new Problem();
```

Comment:
This check must be moved in `base.GetEntityModel(model);` if there is no such a check there.